web/app/js/components/ResourceDetail.jsx

```diff
@@ -86,6 +87,14 @@ export class ResourceDetailBase extends React.Component {
       this.api.fetchMetrics(
         `${this.api.urlsForResource(resource.type, resource.namespace)}&resource_name=${resource.name}`
       ),
+      // upstream resources of this resource (meshed traffic only)
+      this.api.fetchMetrics(
+        `${this.api.urlsForResource(resource.type, resource.namespace)}&to_name=${resource.name}`
+      ),
+      // downstream resources of this resource (meshed traffic only)
+      this.api.fetchMetrics(
+        `${this.api.urlsForResource(resource.type, resource.namespace)}&from_name=${resource.name}`
```

Comment:
I think you'd also need to specify `resource_type=all`, so something like this:
```
http://localhost:8084/api/tps-reports?resource_type=all&from_name=web&from_type=deployment&window=1m&all_namespaces=true
```

Reply:
I've updated the query so that now all we need to do to get metrics for all resource types is switch resourceType to "all". Since we don't display it anywhere I've left it as querying for the same resource type that we're on, but in a subsequent branch I'll probably add some tables that'll use it.